docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -813,6 +814,16 @@ Size of /dev/shm.
 
 This is equivalent to the Podman `--shm-size` option and generally has the form `number[unit]`
 
+### `StartWithPod=`
+
+Start the container aftert the associated pod is created. Default to **true**.
+
+If `true`, container will be started/stopped/restarted alongside the pod.
+
+If `false`, the container will not be started when the pod starts. The container will be stopped when the pod. Restarting the pod will also restart the container.
```

Comment:
```suggestion
If `false`, the container will not be started when the pod starts. The container will be stopped with the pod. Restarting the pod will also restart the container as long as the container was also running before.
```
I think? Because if it was not active before the unit will not be started I would assume?

Reply:
Yeah, that's what I meant. Better be explicit here